include/cantera/kinetics/InterfaceKinetics.h

```diff
@@ -106,6 +106,9 @@ class InterfaceKinetics : public Kinetics
     virtual void getDeltaSSEnthalpy(doublereal* deltaH);
     virtual void getDeltaSSEntropy(doublereal* deltaS);
 
+    virtual void getDeltaRefGibbs(doublereal* deltaG);
+    virtual void getDeltaRefEntropy(doublereal* deltaS);
+
```

Comment:
For consistency add `getDeltaRefEnthalpy` as well.

Reply:
Done